### PR TITLE
Add TruffleRuby 20.3.0

### PR DESCRIPTION
**Pull request by eregon**
cc @havenwood, @pkuczynski

### CODE REVIEW

**Review by pkuczynski — APPROVED** [review]

**Comment by eregon** [comment]
CI is passing, except the first job which was already failing and unrelated:
https://travis-ci.org/github/rvm/rvm/builds/742774481
This means I can't click `Merge pull request`.

I think let's add that job to allow failures since it fails anyway.

**Comment by eregon** [comment]
@pkuczynski Could you re-approve since pushing a new commit removed the approval?

**Comment by pkuczynski** [comment]
Sure!

**Review by pkuczynski — APPROVED** [review]

File: .travis.yml
Comment (pkuczynski) [inline .travis.yml:62]: Why do we want this?

**Review by pkuczynski — CHANGES_REQUESTED** [review]

File: .travis.yml
Comment (eregon) [reply .travis.yml:62]: Otherwise I cannot merge the PR, as I explained in https://github.com/rvm/rvm/pull/5005#issuecomment-729604991.

Comment (eregon) [reply .travis.yml:62]: In case it wasn't clear, that job has been failing [for ages](https://travis-ci.org/github/rvm/rvm/builds), and nobody seems to care to fix it, so there is no point to pretend it passes, and I think we can acknowledge it fails and have the CI result actually reflects if the PR breaks something.
Plus, it prevents PRs for contributors like me with write access.

Comment (pkuczynski) [reply .travis.yml:62]: Make sense!

**Review by pkuczynski — APPROVED** [review]

**Pull request merged**
